Detectors/TPC/workflow/src/CATrackerSpec.cxx

```diff
@@ -538,7 +539,12 @@ DataProcessorSpec getCATrackerSpec(ca::Config const& specconfig, std::vector<int
         ClusterNativeHelper::Reader::fillIndex(clusterIndex, clusterBuffer, clustersMCBuffer, inputs, mcInputs, [&validInputs](auto& index) { return validInputs.test(index); });
         ptrs.clusters = &clusterIndex;
       }
-      int retVal = tracker->runTracking(&ptrs);
+      GPUInterfaceOutputs outputRegions;
+      size_t bufferSize = 2048ul * 1024 * 1024; // TODO: Just allocated some large buffer for now, should estimate this correctly;
+      auto bufferCompressedClusters = pc.outputs().make<std::vector<char>>(Output{gDataOriginTPC, "COMPCLUSTERS", 0}, bufferSize);
```

Comment:
Looks like you are creating a local copy here instead of taking the ref., should be:
``auto& bufferCompressedClusters = pc.outputs().make<std::vector<char>>...``

Reply:
That would explain all my problems, thanks, will check and report back.